services/proxy/src/Proxy/Run.hs

```diff
@@ -0,0 +1,27 @@
+module Proxy.Run (run) where
+
+import Imports hiding (head)
+import Control.Monad.Catch
+import Control.Lens hiding ((.=))
+import Data.Metrics.Middleware hiding (path)
+import Data.Metrics.Middleware.Prometheus (waiPrometheusMiddleware)
+import Data.Metrics.WaiRoute (treeToPaths)
+import Network.Wai.Utilities.Server hiding (serverPort)
+import Network.Wai.Handler.Warp (runSettings)
+import Proxy.Env
+import Proxy.Proxy
+import Proxy.Options
+import Proxy.API (sitemap)
+
+run :: Opts -> IO ()
+run o = do
+    m <- metrics
+    e <- createEnv m o
+    s <- newSettings $ defaultServer (o^.host) (o^.port) (e^.applog) m
+    let rtree    = compile (sitemap e)
+    let measured = measureRequests m (treeToPaths rtree)
```

Comment:
You could add a `-- DEPRECATED` comment for `measured`, the collectd metrics exposed under `/i/monitoring`. (same for other services) Note this can't be removed until the prometheus metrics are deployed to prod and all dashboard have been adapted.

Reply:
(please add it to the function definition, not this call :)